app/src/main/res/values-v21/themes.xml

```diff
@@ -12,7 +12,7 @@
   <style name="OppiaThemeWithoutActionBar" parent="Theme.MaterialComponents.Light.NoActionBar.Bridge">
     <item name="colorPrimary">@color/color_palette_primary_color</item>
     <item name="colorPrimaryDark">@color/color_palette_primary_dark_color</item>
-    <item name="colorAccent">@color/color_palette_accent_color</item>
+    <item name="colorAccent">@color/component_color_shared_edit_text_cursor_color</item>
```

Comment:
This should not be changed here because `colorAccent` must be getting used at various other places too. and it will affect other UI elements when changed.

Changing it in theme is fine but we should be changing the exact `curor_color` to some similar attribute and not its parent attribute like `colorAccent`.

Also the color in lightmode has changed because of latest commit. It should not be black, it should be blue  https://xd.adobe.com/view/3dca36c2-5115-419c-b25e-0f10526b077c-6899/screen/5ef661fa-8995-45b2-b10d-64c1b5cc7890/